.travis.yml

```diff
@@ -0,0 +1,31 @@
+language: python
+
+python:
+  - 2.7
+  - 3.3
+  - 3.4
+
+env:
+  - DEPS="numpy=1.8 matplotlib=1.3 jinja2=2.7.2 pandas=0.13.1 nose"
+
+install:
+  - conda create -n testenv --yes python=$TRAVIS_PYTHON_VERSION
+  - source activate testenv
+  - conda install --yes $DEPS
+  - python setup.py install
+
+before_install:
+  # setup virtual x
+  - "export DISPLAY=:99.0"
```

Comment:
In fact I am curious about these two lines... why you are using them there? I don't think it is necessary, but maybe I am missing something...